src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -37,8 +37,10 @@ RaftReplDev::RaftReplDev(RaftReplService& svc, superblk< raft_repl_dev_superblk
     if (load_existing) {
         m_data_journal =
             std::make_shared< ReplLogStore >(*this, *m_state_machine, m_rd_sb->logdev_id, m_rd_sb->logstore_id);
+        m_data_journal->register_log_found_cb(
+            [this](logstore_seq_num_t lsn, log_buffer buf, void* key) { on_log_found(lsn, buf, key); });
```

Comment:
Can we avoid to register and put it in the constructor itself. I wanted to remove register calls going forward anyways.